tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -21,8 +23,11 @@ def __exit__(self, *args):
 
 class UnaryMathTestBase(object):
 
+    input = None
```

Comment:
This fix is for the tests for array-scalar binary operation, but this class is also used for other functions which take a single array for the argument. How about defining a new 
derived class for array-scalar tests?

Reply:
I don't necessarily understand the point, but `self.input` is already used in thi class. Such refactoring could be done separately.